app/queues/notifications/record-expiration-monitor.worker.ts

```diff
@@ -0,0 +1,53 @@
+import { Worker } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
+
+export const recordExpirationQueueName = 'notifications-recordExpiration';
+
+const updateRecordNotifiedStatus = async (recordId: number) => {
+  return await prisma.record.update({
+    where: {
+      id: recordId,
+    },
+    data: {
+      isNotified: true,
```

Comment:
Question: Where do you set this back to false? When the renew button is pressed?

Reply:
Good catch. I actually forgot we could renew certificates.

Reply:
The scope of this PR is to just just send one month notice. I am not sure if I am supposed to set the value of isNotified to true in this PR.

Reply:
> It depends, If record renew is already complete, then by introducing this field, you should also integrate it to the appropriate logic to set it correctly (on record renew set it to false)
> 
> If record renew is not yet implemented, it will fall on that task to also add this. Not sure what's the state of that functionality

I don't think the record renew feature is fully implemented yet. Please let me know if I am wrong.